Lib/test/test_inspect.py

```diff
@@ -2,6 +2,7 @@
 import collections
 import datetime
 import functools
+from functools import partial
```

Comment:
Please use module import (i.e. `import functools`).

Reply:
Done in adb5db6. I did that originally to simplify the double `functools.partial` wrapping in the tests.